includes/elements/section.php

```diff
@@ -552,6 +552,22 @@ protected function register_controls() {
 				'fields_options' => [
 					'background' => [
 						'frontend_available' => true,
+						'assets' => [
+							'styles' => [
+								[
+									'name' => 'swipper',
```

Comment:
I am not sure if this approach is working?

```suggestion
									'name' => 'swiper',
```